src/embezzler.ts

```diff
@@ -568,6 +570,25 @@ export const gregFights = (
   fightsProp: NumericProperty,
   totalCharges: () => number,
 ) => {
+  function runGregFight(options: EmbezzlerFightRunOptions) {
+    const run = ltbRun();
+    run.constraints.preparation?.();
+    const bunnyBanish = [...getBanishedMonsters().entries()].find(
+      ([, monster]) => monster === $monster`fluffy bunny`,
+    )?.[0];
+    const adventureFunction = options.useAuto ? garboAdventureAuto : garboAdventure;
+    adventureFunction(
+      $location`The Dire Warren`,
+      Macro.if_($monster`fluffy bunny`, run.macro).step(options.macro),
+      Macro.if_($monster`fluffy bunny`, run.macro).step(options.macro),
+    );
+
+    if (get("lastEncounter") === $monster`fluffy bunny`.name && bunnyBanish) {
+      abort(
+        `Fluffy bunny is supposedly banished by ${bunnyBanish}, but this appears not to be the case; the most likely issue is that your ${monsterProp} preference is assigned an incorrect value.`,
```

Comment:
Could be more specific and say it should be set to 0, not just an incorrect value.